spec/unit/file_system_spec.rb

```diff
@@ -134,6 +134,36 @@ def increment_counter_in_multiple_processes(file, num_procs, options)
     end
   end
 
+  context "read should allow an encoding to be specified" do
+    # First line of Rune version of Rune poem at http://www.columbia.edu/~fdc/utf8/
+    # characters chosen since they will not parse on Windows with codepage 437 or 1252
+    let (:rune_utf8) { "\u16A0\u16C7\u16BB" } # 'ᚠᛇᚻ'
+
+    before :each do
+      # verify Ruby has treated string as UTF-8
+      expect(rune_utf8.encoding).to eq(Encoding::UTF_8)
```

Comment:
Yeah, this was the pre-flight check to ensure we were in the right encoding, but I'll switch over to using raw bytes and `pack('c*')` like I have in the past if you don't like this style ;0